include/alpaka/math/abs/AbsStdLib.hpp

```diff
@@ -29,17 +30,15 @@ namespace alpaka
         {
             //! The standard library abs trait specialization.
             template<typename TArg>
-            struct Abs<
-                AbsStdLib,
-                TArg,
-                std::enable_if_t<std::is_arithmetic<TArg>::value && std::is_signed<TArg>::value>>
+            struct Abs<AbsStdLib, TArg>
             {
-                ALPAKA_FN_HOST auto operator()(AbsStdLib const& abs_ctx, TArg const& arg)
+                ALPAKA_FN_HOST auto operator()(AbsStdLib const& abs_ctx, ConvertedArg<TArg> const& arg)
```

Comment:
So the idea for the host side is to convert `alpaka::Complex<T>` into `std::complex<T>` and just call the standard functions like we do for real numbers. Unfortunately, it didn't work as is, since typecast didn't get along with template substitution. To avoid having to specialize for `Complex` separately (which would have been needed for each math function), I've wrapped this type.